docs/pages/loaders.vue

```diff
@@ -11,6 +11,99 @@
       </p>
     </DocsPageSection>
 
+    <DocsPageSection title="Linear loaders" anchor="#linear loaders">
+      <p>
+        Linear loaders illustrate progress by moving an indicator along a horizontal track. There are two types of linear loaders:
+      </p>
+
+      <ul>
+        <li>Determinate: these loaders show a visual indicator of detectable and measurable progress percentage</li>
+        <DocsShow>
+          <KLinearLoader
+            :progress="75"
+            type="determinate"
+            class="loader"
+            style="width: 400px; margin: 24px"
+          />
+        </DocsShow>
+        <li>Indeterminate: these loaders show an ongoing process with an unspecified or undetectable percentage completion</li>
+        <DocsShow>
+          <KLinearLoader
+            :progress="75"
+            type="indeterminate"
+            class="loader"
+            style="width: 400px; margin: 24px"
+          />
+        </DocsShow>
+      </ul>
+
+      <p>
+        <strong>Usages</strong>
+      </p>
+      <ul>
+        <li>Append to appbars to load initial page content</li>
+        <li>Append to data tables to load table content</li>
+        <li>Use as progress bars to represent task percentage progress</li>
+      </ul>
+
+      <p>
+        <strong>Basic anatomy</strong>
+      </p>
+      <ul>
+        <li>Progress track uses palette.grey.v_200</li>
+        <li>Progress indicator uses brand.secondary.v_100</li>
+        <li>Progress track height is 4px</li>
+        <li>Include a 2px corner radius</li>
+        <li>Remove the corner radius when appended to other components, such as appbars or data tables</li>
+      </ul>
+
+      <p>
+        <strong>Placement</strong>
+      </p>
+      <ul>
+        <li>Can be attached to existing components such as appbars or table rows</li>
+        <li>As a progress bar, should be near task-related metadata</li>
+      </ul>
```

Comment:
We should avoid bolded paragraphs being used instead of real headers. A common structure for a section might have a general pattern of:

> ## Header
> 
> Overview informational paragraph. Here are some related things:
> 
> * one thing
> * another thing
>
> Some additional information if necessary